cla_backend/apps/cla_provider/serializers.py

```diff
@@ -283,6 +284,10 @@ class Meta(AdaptationDetailsSerializerBase.Meta):
         )
 
 
+class ContactResearchMethodSerializer(ContactResearchMethodSerializerBase):
```

Comment:
What is the purpose of not just using the `Base` class? (and then renaming it to remove the `Base` from the name if it's always used directly and never subclassed)

Reply:
Good point. I think I was just copying the pattern for having a separate serialiser for the providers but the fields are the same as the call_centre fields so don't need this empty class. Will remove and update.